python/mp/effects/bounce.py

```diff
@@ -10,6 +11,9 @@ class Bounce(effect.Effect):
     self.direction is a positive or negative integer.
     """
 
+    # Wish there were a better way than requiring this every time
+    dm = copy.deepcopy(effect.Effect.dm)
```

Comment:
Blargh, don't have a better way of doing this.

- Decorators are executed on the class, and not instances, so I can't use something like `self.dm` even though effect.Bounce inherits a dm object from effect.Effect
- I tried just directly using effect.Effect.dm, but then all effects end up using the same one, and everyone ends up having the same set of paths
- So...we're left with this nasty kludge